pkg/storage/streams/store.go

```diff
@@ -146,30 +147,30 @@ func (s *streamStore) Get(ctx context.Context, path paths.Path) (
 	msi := streamspb.MetaStreamInfo{}
 	err = proto.Unmarshal(lastSegmentMeta.Data, &msi)
 	if err != nil {
-		_ = lastRangerCloser.Close()
+		utils.LogClose(lastRangerCloser)
```

Comment:
can combine into err using CombineErrors, instead of logging them here

Reply:
In my next PR I am going to remove Close() from the rangers (the connection will be closed by the reader's Close, not the ranger's one), so all these closes will be removed.